tf/chunkparser.py

```diff
@@ -278,10 +379,16 @@ def convert_v5_to_tuple(self, content):
                  self.flat_planes[1]
 
         assert len(planes) == ((8 * 13 * 1 + 8 * 1 * 1) * 8 * 8 * 4)
-        winner = float(winner)
-        assert winner == 1.0 or winner == -1.0 or winner == 0.0
-        winner = struct.pack('fff', winner == 1.0, winner == 0.0,
-                             winner == -1.0)
+
+        # jhorthos query - pls check this
+        if ver == V6_VERSION:
+            winner = struct.pack('fff', 0.5 * (1.0 - result_d + result_q), result_d,
+                              0.5 * (1.0 - result_d - result_q))
+        else:
+            winner = float(dep_result)
+            assert dep_result == 1.0 or dep_result == -1.0 or dep_result == 0.0
+            dep_result = struct.pack('fff', dep_result == 1.0, dep_result == 0.0,
```

Comment:
On this line the first dep_result should be winner - since its what is returned.